pyrios/main.go

```diff
@@ -26,7 +26,7 @@ import (
 	"cloud.google.com/go/compute/metadata"
```

Comment:
can you format these.  We need the formatter we use in kops

Reply:
my bad, when testing is hooked up, we'll catch that :)